app/components/public/call-for-speakers.js

```diff
@@ -13,6 +13,18 @@ export default class CallForSpeakers extends Component {
     }
   }
 
+  @action
+  addSession() {
+    if (this.data.userSpeaker && this.data.userSpeaker.toArray().length) {
+      // speaker detail exists
```

Comment:
@iamareebjamal Should I add `this.transitionToRoute` here for directing towards `public.cfs.new-session` route?

Reply:
Yes